pkg/chart/chart.go

```diff
@@ -280,17 +280,22 @@ func (t *Testing) LintChart(chart string, valuesFiles []string) TestResult {
 	chartYaml := path.Join(chart, "Chart.yaml")
 	valuesYaml := path.Join(chart, "values.yaml")
 
-	if err := t.linter.Yamale(chartYaml, t.config.ChartYamlSchema); err != nil {
-		result.Error = err
-		return result
-	}
-	if err := t.linter.YamlLint(chartYaml, t.config.LintConf); err != nil {
-		result.Error = err
-		return result
+	if !t.config.NoChartSchemaValidation {
+		if err := t.linter.Yamale(chartYaml, t.config.ChartYamlSchema); err != nil {
+			result.Error = err
+			return result
+		}
 	}
-	if err := t.linter.YamlLint(valuesYaml, t.config.LintConf); err != nil {
-		result.Error = err
-		return result
+
+	if !t.config.NoYamlLint {
```

Comment:
Reflecting about this again, I'm think it would be better to do it the other way around an call the flags `validate-yaml` and `validate-chart-schema`. This would also be more consistent with the existing flags.

Reply:
sure, was hard to me to think in the negative 😄 i will update
thanks for the review @unguiculus

Reply:
done